src/panels/lovelace/cards/hui-error-card.ts

```diff
@@ -43,7 +43,7 @@ export class HuiErrorCard extends LitElement implements LovelaceCard {
     return css`
       :host {
         display: block;
-        background-color: #ef5350;
+        background-color: var(--error-color);
```

Comment:
Is this the same color?

Reply:
Default is `#E45E65`
![image](https://user-images.githubusercontent.com/1299821/93019585-40f8a700-f5d8-11ea-8b62-09b93e00a005.png)

Reply:
Should it be updated in the place where error color is defined, or is this a bad idea?

Reply:
IIRC the text color is white on the error card, that might be a bit low contrast with the new color, but I'm not sure what I think. I like there being fewer colors and more variables...
![image](https://user-images.githubusercontent.com/1299821/93019817-b87b0600-f5d9-11ea-98e7-23d4863a202d.png)

Reply:
I think it is ok... I would like to have just 1 error color